framework_tvb/tvb/interfaces/web/templates/jinja2/form_fields/datatype_select_field.html

```diff
@@ -1,13 +1,30 @@
-<p class="field-data">
-<select name="{{ field.name }}" id="{{ field.name }}" class="dataset-selector" tabindex="1">
+<script type="text/javascript" src="/static/js/filters.js?{{ currentVersionJS }}"></script>
```

Comment:
Can't you put the import of `filters.js` on a higher level, e.g. in other jinja template where `datatype_select_field.html` gets included also?
If you put it here, it will be included multiple times, as we might have the fragment `datatype_select_field.html` repeating on the same page.